.github/workflows/build_and_test.yml

```diff
@@ -132,7 +138,7 @@ jobs:
         env:
           METEOR_PROFILE: 1000
         run: |
-          meteor build --server-only --directory --debug /tmp/build-test
+          meteor build --server-only --directory --debug /tmp/build-test --verbose
```

Comment:
Did we learn anything using "--verbose"?  Do we want to keep it?

Reply:
Nothing at all. It logged only downloading dependencies and extracting them. Nothing useful.